crates/sui-core/src/transaction_driver/effects_certifier.rs

```diff
@@ -569,6 +573,7 @@ impl EffectsCertifier {
         client: &Arc<SafeClient<A>>,
         client_monitor: &Arc<ValidatorClientMonitor<A>>,
         raw_request: &RawWaitForEffectsRequest,
+        ping: Option<PingType>,
```

Comment:
Maybe I just removed this recently. Can you get this value from `raw_request`?

Reply:
I believe I can do. Let me confirm and update.